### DISCUSSION TITLE
Creator Coin Subscriptions

### DISCUSSION THREAD

**Discussion by 1dolinski** [deso-protocol/dips#204] [Large CIPs]
Hey all,

### Summary - A Subscription Primitive with Monthly Distributions 
Having on-chain Creator Subscriptions would be notably beneficial for creators, coin holders, the community, and DeSo as a whole. The creator coin and bonding curve were genius inventions, yet accurately valuing a creator coin at the moment is unsolved and coin holder benefits are not clear. We subscribe to Netflix for content, buying its stock or creator coin serves a different purpose. 

Creator coin subscriptions would provide an easy way to:
- allow creators to offer premium content and earn a stable cash flow
- allow fans / customers to align their interests and support a creator by subscribing to their premium content. 
- allow coin holders earn alongside the creator as as portion of the subscription could be paid out to the coin holders
- clearly model the floor value of a Creator Coin using a Discounted Cash Flow model
- align Creator and Coin Holder incentives, both win when more people subscribe
- solve the Rug Pull problem as selling below the valuation price would be quickly bought up
- subscriptions are the most common way to pay for recurring content.

It does a lot, but we can think of the simple version as a button [ Subscribe to Creator for 50 cents a month ], a premium feed, and monthly distributions to the creator and coin holders.

This idea isn't new and was written about in the original white-paper:
```Thus, people who believe in someone’s potential can buy their coin and succeed with them financially when that person realizes their potential. And traders can make money buying and selling the ups and downs.```

### Subscriptions Are Widely Used Outside of DeSo
There are broadly 3 types of ways to use DeSo and dollars
1. Purchase - one-time transaction
2. Investment - owning a piece of something as its value goes up and down
3. Subscription - getting continued access to content and services for a recurring fee as they are provided 

Purchases and investment functionality generally works well with NFTs and CCs. NFTs provide subscription like functionality but are missing the recurring portion and are not ubiquitous. NFTs are not a subscription primitive. Creators can't be sure when the next sale of their NFT will occur so the distributions are inconsistent.

Functionally, buying and selling a CC works very well. It's not an investment at the moment because there isn't an easy way to value it.

Subscriptions for content / service access are found everywhere but not on DeSo: Patreon, Netflix, Phone plans, Gym Memberships. We buy them for continued access and our society is very familiar with them.

### How It Works by User Type
**Fans *and* customers of a creators content** - They would subscribe for premium access. 10 cents/mo, $1/mo, $20/mo. Apps could be spun up around this but the payment portion and distribution needs to be **on the chain or done by a trusted smart service**.

**People who hold the CC** - they would get a portion of the subscription. It would be ideal if creator could set this percentage, perhaps it's the same as the FR. They are then benefiting from the growth in subscribers as well. They would be aligned and incentivized to help promote subscriptions of the creator.

**Creators** - they would get the residual percentage of the subscription.

### Premium Feed
Each creator would have a "Premium feed" which would sit beside the "Following, Global, NFTs, [Premium]". This is the MVP implementation, by itself, it's massive. There are plenty of apps that can further be created on the back of an on-chain subscription.

### DeSo Unique Advantages
**Importantly!** Because of its low transaction costs, DeSo is uniquely able to create tiny subscriptions! This is amazing and a total game changer. The industry standard Stripe charges 2.9% + 30c on top of the platform charges!! DeSo literally can do things on-chain that others cannot.

Other platforms, such as Instagram in the screenshot below, have recently added subscriptions but they are different than DeSo because of DeSo's native coin holder functionality. It's special. Their version can be thought of a 2 coin holder subscription. The creator and the platform. DeSo allows for many to join in and promote creators. The bonding curve shape specifically helps incentivize people to buy the CC early, it is designed to be gas on the fire for every creator.

### Rugpuller's Have a Reason to Think Twice
This also solves the notorious "Rug pull". In the investing world we have dividends and cash flows which are used as the main input to model most companies through the Discounted Cash Flow model (https://www.investopedia.com/terms/d/dcf.asp). It says a coin is worth the value of its discounted cash flows. Here a Creator Coin would be the discounted future subscription amounts. We could use this same model to value creator coins. I'm in favour of people being able to buy and sell coins, whenever they want. I don't think coin locks is a viable solution. With creator coin valuations enabled by subscriptions, people will gladly buy up a coin if it's below it's DCF value. 

### Pull New Users into DeSo
**Subscriptions will pull in new users** as DeSo nodes are the **only place** they would be able to get access to the content. Imagine Tropix offering his 400k subscribers on TikTok an option to subscribe to his premium content for 25 cents or a dollar a month. He would collect sizeable earnings and his passionate fans that own his coin can join him in seeing the benefit of his other fans subscribing.

By enabling CCs to have value we will extend and further realize a huge piece of what was **originally outlined in the white-paper**. When creators grow and succeed, their coin holders and loyal followers will as well. Truly amazing. 

### Why Subscriptions Should Not Be On an Individual Node Level
We commonly subscribe to Netflix, Crave, Disney Plus, and Apple TV separately. This isn't ideal as the consumer but we deal with it because they are big companies and importantly, they offer different content. Subscribing to the same creator on different nodes would be like subscribing to these platforms and them providing the same content.

### Related posts on-chain
https://bitclout.com/posts/3b2e5e3ee00b56a494ae4521add8bfb05aae5a9185b0eec6090042fb69b37a96?tab=posts
https://bitclout.com/posts/a80546d8c8575a1f0ae86832d73fa8191a74bd4c36dd145cd2333d0b8c8b2cd4?tab=posts
https://bitclout.com/posts/959c436138bfc18bec087fa594e5e95f10d2dd2b4b54ac84c7661534646df24a?tab=posts
https://bitclout.com/posts/cd0997cbf4427b52c585a558055404c3734202c3b7fe08d892e2dce53200d746?tab=posts

Daily DIP 1 - https://bitclout.com/posts/80529fa62734eab1dd1979c4eb01000dd0af00c5e177e06d1bfc79820c61f52b?feedTab=Following
Daily DIP 2 - https://bitclout.com/posts/bb30e0fa063fe97c51edc34b27fa9bd46133c68d04b6a107f0abaf8d113b3a33?feedTab=Following

Recently Instagram has decided to get into this as well.
Image: ![Instagram Subscriptions](https://media.discordapp.net/attachments/867153094876725268/938915407056764968/unknown.png "Instagram Subscriptions")
Post: https://bitclout.com/posts/f877e753519999ab7683cc5f7e27271854fac343a2861b0eb4a201496f70db66?tab=posts

### Ways this Can be Done (in progress)
The post above outlined why this is a good thing. As we know yes is nothing without how. There are a few options to consider, each has its own set of pros and cons.

### Additional Discussions (in progress)
Any regulatory thoughts, ways to move this forward

### Additional Coverage
<img width="600" alt="image" src="https://user-images.githubusercontent.com/1243216/171879474-51f6cc2c-99c0-4f98-a3ea-188051f18dc7.png">

https://twitter.com/dharmesh/status/1532725567765757953

Update: Feb 4.
Discussion about the "privateness" of on-chain content. 
https://bitclout.com/posts/985206e818854981c3d945e2c8409b9438fcee33c0ae3e50fbf9fe579cb42d0a?tab=posts

**Comment by arhebbar** [comment]
Great post. Other apps that have tried to do this off-chain: [Bitperks/Agoralabs](https://www.agoralabs.xyz/), [Cloutpub](http://cloutpub.org/), [Cuprite](https://cuprite.org/) and even DeSoNocode seems to be trying to do it. Definite plus in getting this as a core feature in terms of standardization. 

Another plus that I see is the fact that the content then doesn't become dependent on centrally controlled apps that if they go down, all the content is lost. e.g. What is Thorsten decides to stop development on Cloutpub?

**Reply by 1dolinski** [reply]
Hey, thanks for your kind words and great comment! 

Those are all great apps. Many of these apps offer access walls based on existing primitives (CC holdings, NFTs) which are already on-chain, it's a great direction. Apps extending primitives is desirable.

These options are different than this DIPs proposed subscription primitive and recurring coin holder distributions.

**Comment by kuririn1** [comment]
I agree, subscriptions would be a huge added value for the chain and could create many new possibilities. It's one of the highest leverage features that could be implemented. 

It would also require content encryption/decryption and revoking access after the subscription is expired (features already used in private messages and derived keys). This could be another very powerful primitive itself.

**Reply by oO** [reply]
I think the Messages V3 infrastructure can support the subscriber-only post access, by having one of many group which are auto-populated based on the subscription status. What would be nice is:

- Ability for only the owner of the group to create posts ( others can only comment )
- Ability for a post to appear in multiple groups ( most subscription models allow for multiple tiers, which each tier being a super-set of the tier below. ( 0.1 deso/mo gets all the content from 0.05 deso/mo + exclusive content )

**Comment by oO** [comment]
See Patreon as an excellent example of a subscription-based creator economy ecosystem.

**Reply by eCentricThumb** [reply]
great if you already have a following. not good for small time beginner creators. 
managing subscriptions is problematic, too.

**Reply by 1dolinski** [reply]
why is it not good for them? why is managing subscriptions problematic?

**Comment by osontungo** [comment]
Why is it that when I read your article, I am more excited about deso, a model of social networking combined with economy

**Comment by safiyamuuse99-alt** [comment]
_waxaan ahay wìl daa yar_

**Comment by safiyamuuse99-alt** [comment]
Maga caygu waa cali

**Comment by safiyamuuse99-alt** [comment]
Jaamac

**Comment by safiyamuuse99-alt** [comment]
Xasan

**Comment by safiyamuuse99-alt** [comment]
`[](url)`ybb vv

> hhjb bbn

**Comment by safiyamuuse99-alt** [comment]
7
